### PR TITLE
Fix bug-1521601: [checks] Enable custom ("pontoon") checks to skip the DB

### PR DESCRIPTION
**Pull request by aniruddha2000**

### CODE REVIEW

File: pontoon/checks/libraries/not_stored.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    """
+    Group all checks related to the base UI
+    :arg pontoon.base.models.Entity entity: Source entity
+    :arg basestring string: a translation
+    """
+    checks = defaultdict(list)
+
+    # Prevent empty translation submissions if supported
+    if string == '' and entity.resource.allows_empty_translations:
+        checks['pWarnings'].append(
```
Comment (mathjazz) [inline pontoon/checks/libraries/not_stored.py]: `nsWarnings`

File: pontoon/checks/libraries/__init__.py [hunk 2]
```diff
@@ -25,7 +25,7 @@ def run_checks(
         * JsonResponse - If there are errors
         * None - If there's no errors and non-omitted warnings.
     """
-    pontoon_checks = pontoon.run_checks(entity, string)
+    pontoon_checks = stored.run_checks(entity, string)
```
Comment (mathjazz) [inline pontoon/checks/libraries/__init__.py]: You also need to add a line to run the `not_stored` checks.

File: pontoon/base/static/js/translate.js [hunk 3]
```diff
@@ -2796,6 +2796,10 @@ var Pontoon = (function (my) {
         this.showFailedCheckType('warning', failedChecks.clWarnings);
       }
 
+      if (failedChecks.pWarnings) {
+        this.showFailedCheckType('warning', failedChecks.pWarnings);
```
Comment (mathjazz) [inline pontoon/base/static/js/translate.js]: `nsWarnings`

File: pontoon/checks/__init__.py [hunk 4]
```diff
@@ -10,6 +10,6 @@
 
 # Warnings and Errors from these libraries will be saved in the DB
 DB_LIBRARIES = (
-    'p',
+    'stored',
```
Comment (mathjazz) [inline pontoon/checks/__init__.py]: `s`

File: pontoon/checks/libraries/not_stored.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
```
Comment (mathjazz) [inline pontoon/checks/libraries/not_stored.py]: I think we should call this file `no**n**_stored.py` instead.

**Comment by mathjazz** [comment]
@aniruddha2000 Let's sync on IRC before making further changes here. Maybe we should just keep "pontoon" and add "pontoon_non_stored" to make the changeset minimal?

File: pontoon/checks/libraries/not_stored.py (hunk 5 above)
Comment (aniruddha2000) [reply pontoon/checks/libraries/not_stored.py]: ok

**Pull request merged**